buildtools/npm/manifest_test.go

```diff
@@ -0,0 +1,68 @@
+package npm
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	// "github.com/fossas/fossa-cli/buildtools/npm/manifest"
+)
+
+func TestFromManifest(t *testing.T) {
+	manifest, err := FromManifest("testdata/package.json")
+	assert.NoError(t, err)
+
+	assert.NotEmpty(t, manifest.Dependencies)
+	assert.Equal(t, manifest.Dependencies["chai"], "4.1.2")
+}
+
+func TestFromNodeModules(t *testing.T) {
+	t.Skip("not yet implemented")
+	manifests, err := FromNodeModules("testdata/package.json")
+	assert.NoError(t, err)
+
+	/*
+		└─┬ chai@4.1.2
+			├── assertion-error@1.1.0
+			├── check-error@1.0.2
+			├─┬ deep-eql@3.0.1
+			│ └── type-detect@4.0.8 deduped
+			├── get-func-name@2.0.0
+			├── pathval@1.1.0
+			└── type-detect@4.0.8
+	*/
+
+	assert.NotEmpty(t, manifests)
+	assert.True(t, containtsDep(manifests, "assertion-error", "1.1.0"), "Manifests does not include dep assertion-error")
+	assert.True(t, containtsDep(manifests, "check-error", "1.0.2"), "Manifests does not include dep check-error")
+	assert.True(t, containtsDep(manifests, "get-func-name", "2.0.0"), "Manifests does not include dep get-func-name")
+	assert.True(t, containtsDep(manifests, "pathval", "1.1.0"), "Manifests does not include dep pathval")
+	assert.True(t, containtsDep(manifests, "type-detect", "4.0.8"), "Manifests does not include dep type-detect")
+
+	// check transitive dep's existance
+	deps, err := selectDep(manifests, "deep-eql", "3.0.1")
+	assert.NoError(t, err, "Manifests does not include dep deep-eql")
+
+	deepEql := deps[0]
+	assert.NotEmpty(t, deepEql.Dependencies)
+	assert.Len(t, deepEql.Dependencies, 1)
+	assert.Contains(t, deepEql.Dependencies, "type-detect")
+}
+
+func selectDep(manifests []Manifest, name string, version string) ([]Manifest, error) {
```

Comment:
This return type should be `(Manifest, error)` given that it only ever returns 1-length slices.